### ISSUE TITLE
Picard tools usage print HTML and poorly formatted at the console

### ISSUE THREAD

**Issue by tfenne** [broadinstitute/picard#409]
It looks like @vdauwera's much needed and appreciated revamp of the tool documentation had an unintended side effect.  Now when I run any Picard tool at the command line the usage prints out with a lot of embedded HTML characters, and little to no formatting.

Here's an example from CollectAlignmentSummaryMetrics:

```
$ java -jar dist/picard.jar CollectAlignmentSummaryMetrics
ERROR: Option 'INPUT' is required.

USAGE: CollectAlignmentSummaryMetrics [options]

Documentation: http://broadinstitute.github.io/picard/command-line-overview.html#CollectAlignmentSummaryMetrics

Produce a summary of alignment metrics from a SAM or BAM fileUsing read outputs from high throughput sequencing (HTS) technologies, this tool provides metrics regarding the quality of read alignments to a reference sequence, as well as the proportion of the reads that passed machine signal-to-noise threshold quality filters (Illumina).<h4>Usage example:</h4><pre>    java -jar picard.jar CollectAlignmentSummaryMetrics \<br />          R=reference_sequence.fasta \<br />          I=input.bam \<br />          O=output.txt</pre>Please see <a href='http://broadinstitute.github.io/picard/picard-metric-definitions.html#AlignmentSummaryMetrics'>the AlignmentSummaryMetrics documentation</a> for detailed explanations of each metric. <br /> <br />Additional information about Illumina's quality filters can be found in the following documents on the Illumina website: <ul><li>http://support.illumina.com/content/dam/illumina-marketing/documents/products/technotes/hiseq-x-percent-pf-technical-note-770-2014-043.pdf</li> <li>http://support.illumina.com/content/dam/illumina-support/documents/documentation/system_documentation/hiseqx/hiseq-x-system-guide-15050091-d.pdf</li></ul><hr />
Version: 2.0.1(e43c89783d810ad663d0d3b7da7adf03196e441d_1452613799)
...
```

I think one of two things needs to happen:
1) Go through the tools again and remove HTML and replace with actual line breaks etc.
2) Put some tooling into Picard so that the HTML is processed out and key elements like `<BR>` translated into line breaks.

**Comment by vdauwera** [comment]
Oh crap. I forgot to take commandline usage into account -- too used to the GATK docs system.

  I would favor the second option because it gives us more flexibility for formatting the documentation web pages.

**Comment by vdauwera** [comment]
@cmnbroad Will Barclay address this? If not already, how feasible would it be to achieve that?

**Comment by vdauwera** [comment]
Oh, actually the grossest symptoms were addressed in #437. There was a request for further improvement of the cleanup function in #439 but it's low priority. 

We should just make sure this isn't an issue with Barclay.
